src/routers/secure/scrape.py

```
@@ -770,806 +679,282 @@ def check_at_least_one_id_provided(self) -> Self:
+    ranking_overrides: dict[str, Any] | None = None
 
 
 async def perform_season_scrape(
     tmdb_id: str | None = None,
     tvdb_id: str | None = None,
     imdb_id: str | None = None,
     season_numbers: list[int] | None = None,
-) -> dict[str, Stream]:
-    """Helper to perform season scraping with state management."""
-
+    ranking_overrides: dict[str, Any] | None = None,
```

Ruff (0.14.10)

Unused function argument: `ranking_overrides`

(ARG001)